eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/service/RequestUriLogger.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2017 SchedJoules
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.schedjoules.eventdiscovery.service;
+
+import android.util.Log;
+
+import org.dmfs.httpessentials.client.HttpRequest;
+import org.dmfs.httpessentials.client.HttpRequestExecutor;
+import org.dmfs.httpessentials.exceptions.ProtocolError;
+import org.dmfs.httpessentials.exceptions.ProtocolException;
+import org.dmfs.httpessentials.exceptions.RedirectionException;
+import org.dmfs.httpessentials.exceptions.UnexpectedStatusException;
+
+import java.io.IOException;
+import java.net.URI;
+
+
+/**
+ * {@link HttpRequestExecutor} decorator that logs request uris to logcat.
+ *
+ * @author Gabor Keszthelyi
+ */
+public class RequestUriLogger implements HttpRequestExecutor
```

Comment:
Please rename to `RequestUriLogging`. This is a decorator and it should follow our decorator naming scheme, i.e. the decorator name describes how it modifies the original behavior. It's not an "-er" type of class.